lib/resources.js

```diff
@@ -60,9 +64,10 @@ exports.categories = {
             url: 'https://github.com/youhusam/hanx.js',
             description: 'Full-stack boilerplate with Node.js, hapi, PostgreSQL and AngularJS (MEAN.js port)'
         },
-        'hapi-api': {
-            url: 'https://github.com/rjmreis/hapi-api',
-            description: 'Lean hapi API Boilerplate with an opinionated view on project structure'
+        'start-hapiness' : {
+            url: 'http://github.com/thebergamo/start-hapiness',
+            description: 'A Boilerplate that help you to create fast project with Hapi + MongoDB'
+        }
         }
```

Comment:
Sorry for that :(


On Fri, Dec 18, 2015 at 3:49 PM Nathan LaFreniere <notifications@github.com>
wrote:

> In lib/resources.js
> <https://github.com/hapijs/hapijs.com/pull/307#discussion_r48050468>:
>
> > @@ -60,9 +64,10 @@ exports.categories = {
> >              url: 'https://github.com/youhusam/hanx.js',
> >              description: 'Full-stack boilerplate with Node.js, hapi, PostgreSQL and AngularJS (MEAN.js port)'
> >          },
> > -        'hapi-api': {
> > -            url: 'https://github.com/rjmreis/hapi-api',
> > -            description: 'Lean hapi API Boilerplate with an opinionated view on project structure'
> > +        'start-hapiness' : {
> > +            url: 'http://github.com/thebergamo/start-hapiness',
> > +            description: 'A Boilerplate that help you to create fast project with Hapi + MongoDB'
> > +        }
> >          }
>
> you have a syntax error here with an extra }
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hapijs/hapijs.com/pull/307/files#r48050468>.
>